gendiff/diff.py

```diff
@@ -0,0 +1,63 @@
+from typing import Any, Dict, Set
+
+from gendiff.file_operations import parse_file
+from tests.paths import resolve_path
+
+
+def get_keys_groups(
```

Comment:
let's update this function to return mapping of key -> change type (e.g. {"kek": "missing_nested"}), so we'll just iterate over it in `get_diff_view`

Reply:
@AlexMomot-717 closing this one as this function will be rewritten